web/static/js/components/respondents/RespondentIndex.jsx

```diff
@@ -423,6 +435,7 @@ class RespondentIndex extends Component<Props, State> {
                 respondent={respondent}
                 responses={responses}
                 variantColumn={variantColumn}
+                surveyModes={survey.mode[0]}
```

Comment:
why a single mode is passed as `surveyModes`? if the respondent used another mode of the survey what is been shown?

Reply:
In flow types Survey.mode in specified as `mode: string[][] `. In a survey that uses IVR and SMS modes, mode is persisted as [["sms","ivr"]]. For some reason I ignore, mode is declared as a multidimensional array.